src/components/Footer/Footer.styles.js

```diff
@@ -15,6 +16,14 @@ export const FooterLinkContainer = styled.span`
   font-size: 14px;
 `;
 
+export const StyledA = styled.a`
+  color: ${({ theme }) => theme.colors.primary};
+`;
+
+export const StyledLink = styled(Link)`
```

Comment:
You can use only the Link here and the styled-components [as](https://styled-components.com/docs/api#as-polymorphic-prop) prop. 

I'm never sure how to name styled components when having naming conflicts btw (like with the Link component here).  It doesn't feel right to prefix everything with `StyledX`, only prefixing some seems inconsistent. What I often did was renaming the import like 

```
import { Link as RawLink } from 'react-router-dom';
```

This way you at least have consistent exports. But the `RawX` prefix also feels stupid... 

Fyi I did some research and found [this nice article](https://medium.com/inturn-eng/naming-styled-components-d7097950a245) about namespacing styled components.

Reply:
This didn't feel right to me as well. I [read a discussion](https://spectrum.chat/styled-components/general/how-do-you-all-handle-name-collision~d438c6d7-e0f2-4394-84e4-a1fb5a812cba) and one on [reddit](https://www.reddit.com/r/reactjs/comments/d7hj9r/how_to_name_styled_components/) as well and decided that Styled looks easier for the one case of naming collision I have here. I will use the RawLink method here but will think about implicit imports for my future projects.

Reply:
Thanks for the links. I'll have a look